mlir/test/IR/properties.mlir

```diff
@@ -2,31 +2,31 @@
 // # RUN: mlir-opt %s -mlir-print-op-generic -split-input-file  | mlir-opt -mlir-print-op-generic | FileCheck %s --check-prefix=GENERIC
 
 // CHECK:   test.with_properties
-// CHECK-SAME: <{a = 32 : i64, array = array<i64: 1, 2, 3, 4>, b = "foo"}>
+// CHECK-SAME: <{a = 32 : i64, array = array<i64: 1, 2, 3, 4>, b = "foo"}>{{$}}
```

Comment:
Is this like ensuring end-of-line matching? I didn't know we had this :)

Reply:
Yes. It's simply that anything surrounded by double braces is a regular expression in FileCheck.